src/EmployerAan/SFA.DAS.EmployerAan.Api.UnitTests/Controllers/CalendarsControllerTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using SFA.DAS.EmployerAan.Api.Controllers;
+using SFA.DAS.EmployerAan.Application.Calendars.Queries.GetCalendars;
+using SFA.DAS.EmployerAan.Models;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.EmployerAan.Api.UnitTests.Controllers;
+public class CalendarsControllerTests
+{
+    [Test]
+    [MoqAutoData]
+    public async Task And_MediatorCommandSuccessful_Then_ReturnOk(
```

Comment:
Test naming convention is TargetMethodName_Condition_Outcome

Reply:
got it. done